src/components/Events/EventSignUpForm.jsx

```diff
@@ -456,8 +456,8 @@ export default function EventSignUpForm({
                       </div>
                     )}
                     <small className="form-text text-muted">
-                      Please upload a screenshot of your payment. Payments can
-                      be sent via:
+                      Please upload a screenshot of your <span style={{ fontWeight: 'bold' }}>$10 online RSVP payment.</span><br />
```

Comment:
i think we'll want to make this dynamic in the future since it's a reusable component.

Reply:
Got it!